python/CMakeLists.txt

```diff
@@ -80,15 +81,6 @@ if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/spglib/_version.py)
     configure_file(_version.py.in spglib/_version.py)
 endif ()
 
-# Copy all python packages to the build directory
-file(COPY spglib
-        DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/)
```

Comment:
This one would still be needed so that the `PYTHONPATH` override pointing to `${CMAKE_CURRENT_BINARY_DIR}/` could work. Although, I don't know if we could just mix them, or if one takes precedence.